client/src/App.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
-import Menu from './MenuTable.js';
-import Inven from './invenTable.js';
+import InvenInsert from './InvenInsert.js';
```

Comment:
Name these in full form please, "InventoryInsert"

Reply:
changes recommended for all the column name are fixed and code is commited.

Reply:
Recommended changes are committed.